Source/engine/demomode.cpp

```diff
@@ -75,6 +75,36 @@ bool Timedemo = false;
 int RecordNumber = -1;
 bool CreateDemoReference = false;
 
+// These options affect gameplay and are stored in the demo file.
+struct {
+	uint8_t tickRate = 20;

```

Comment:
isn't this more of a playback speed, we do allow changing it during playback

Reply:
Done